manifests/input.yaml

```diff
@@ -652,27 +647,27 @@ work/input/make_ryf/make_ryf.py:
     binhash: 9fbb6a9f6bec50ccb2afafecae9a5ff7
     md5: c5ef2aeff9516ec76c80e76aef08d352
 work/input/ocean_hgrid.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/ocean_hgrid.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/mom/ocean_hgrid.nc
   hashes:
-    binhash: 7f9cf6930cda20f2ba304eef45c2a97d
-    md5: 51f58be0f4ea6da2cb438a893f95c689
+    binhash: e0b2f3bb2d76badb35e2dfc07788738f
+    md5: 38b6f324ae16cc13c180699123ad85b5
 work/input/ocean_temp_salt.res.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/ocean_temp_salt.res.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/mom/ocean_temp_salt.res.nc
   hashes:
-    binhash: 4812809ecd2abcc47351263cd2c66821
-    md5: c5f7e60b5427a4442f111adc65a2d067
+    binhash: 1c27f9fbe2c7755e4657ed14e5e14145
+    md5: 1797b25e6339595a320076080145fd5a
 work/input/ocean_vgrid.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/ocean_vgrid.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/mom/ocean_vgrid.nc
   hashes:
-    binhash: 30af5058d31d9133770757144f6bd7cf
-    md5: a1e0b0b0adc363506ff17a362e83f64a
+    binhash: a10a26a7c361279ae0032afc8c83fadd
+    md5: e56e2936499d3ad53429d5940821118d
 work/input/salt_sfc_restore.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/mom/salt_sfc_restore.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/mom/salt_sfc_restore.nc
   hashes:
-    binhash: d1546a2b957a23e34f1154ad180df60d
-    md5: b2bd35c44017597ba99b85fb61ed4d72
+    binhash: abaea44efc3b0b31ab1a8eb9f46da744
+    md5: 434b4bdb323d17e9714d02519627fea2
 work/input/topog.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/share/topog.nc
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/share/topog.nc
```

Comment:
There is also a `topog.nc` file in `/g/data/ik11/inputs/access-om3/0.x.0/025deg/mom`. This should be removed

Reply:
I've removed this